modules/sso.php

```diff
@@ -1002,8 +1020,16 @@ public function error_msg_email_already_exists( $message ) {
 	 * @return string
 	 **/
 	public function msg_login_by_jetpack( $message ) {
-
-		$msg = __( sprintf( 'Jetpack authenticates through WordPress.com — to log in, enter your WordPress.com username and password, or <a href="%1$s" target="_blank">visit WordPress.com</a> to create a free account now.', 'http://wordpress.com/signup' ) , 'jetpack' );
+		$msg = sprintf(
+			wp_kses(
+				__(
```

Comment:
Jetpack requires a WordPress.com account. Enter your WordPress.com username and password, or <a href="%1$s" target="_blank">visit WordPress.com</a> to create a free account now.